pkg/installer/config.go

```diff
@@ -49,10 +52,10 @@ func defaultIfZeroString(left, right string) string {
 type Config struct {
 	Identity string
 
-	// DirectCSIContainerImage properties
-	DirectCSIContainerImage    string
-	DirectCSIContainerOrg      string
-	DirectCSIContainerRegistry string
+	// DirectPVContainerImage properties
```

Comment:
Is this comment needed?

Reply:
keeping the comment as it was there already @wlan0